### PR TITLE
Add an API endpoint to get information about data dumps

### PR DESCRIPTION
**Pull request by paramsingh**
This endpoint is gonna be useful for getting information
about dumps when we need to download and import them
into the spark cluster.

### CODE REVIEW

File: listenbrainz/webserver/views/dump_api.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from flask import Blueprint, request, jsonify
+from listenbrainz.webserver.errors import APIBadRequest, APINotFound
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+import listenbrainz.db.dump as db_dump
+
+dump_api_bp = Blueprint("dump_api_v1", __name__)
```
Comment (mayhem) [inline listenbrainz/webserver/views/dump_api.py]: Hmm, I'm not a fan of that name. I think we should think bigger and make a more generic "health-stats" or some such thing that doesn't narrow the scope too much.

File: listenbrainz/webserver/views/dump_api.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    **Example response**:
+
+    .. sourcecode:: json
+        {
+            "id": 1,
+            "timestamp": "20190625-165900"
+        }
+
+    :query id: *Required.* Integer specifying the ID of the dump
```
Comment (mayhem) [inline listenbrainz/webserver/views/dump_api.py]: What if I don't have one? If not specified, I think we should return the date and id of the latest dump...

Comment (paramsingh) [reply listenbrainz/webserver/views/dump_api.py]: I was returning 400 if there wasn't an ID provided, I've changed it now to return the latest.

File: listenbrainz/webserver/views/dump_api.py (hunk 1 above)
Comment (paramsingh) [reply listenbrainz/webserver/views/dump_api.py]: Done after IRC discussion!

**Pull request merged**